unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesAPIController.cs

```diff
@@ -40,6 +52,16 @@ public void GetAllFavorites(Action<List<HotSceneInfo>> OnCompleted)
         OnCompleted?.Invoke(new List<HotSceneInfo>());
     }
 
+    public void AddFavorite(string placeUUID)
+    {
+        //TODO: wire the request once kernel part is ready
+    }
+
+    public void RemoveFavorite(string placeUUID)
+    {
+        //TODO: wire the request once kernel part is ready
+    }
+
```

Comment:
I would remove noisy //TODO together with this functions and add "wire Add/Remove favorite requests once kernel part is ready" as a requirement to the related ticket